Vagrantfile

```diff
@@ -57,22 +57,21 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.define "tester-win12-64" do |testvm|
 	     testvm.vm.box = "https://s3.amazonaws.com/beats-files/vagrant/beats-win2012-r2-virtualbox-2015-12-10_1222.box"
 
-	testvm.ssh.port = 2406
-	testvm.vm.network "forwarded_port", guest: 22, host: testvm.ssh.port, host_ip: "127.0.0.1"
-	testvm.vm.network "private_network", ip: "192.168.33.75"
-
-	testvm.vm.communicator = "winrm"
-	testvm.vm.network "forwarded_port", host: 3389, guest: 3389, host_ip: "127.0.0.1"
-	testvm.vm.network "forwarded_port", host: 5985, guest: 5985, host_ip: "127.0.0.1"
-
-	testvm.vm.provider "virtualbox" do |v|
-	    v.gui = false
-	    v.cpus = 2
-	    v.memory = 2048
-	end
+	      testvm.ssh.port = 2406
```

Comment:
It looks like the indentation is off here or is it just Github diff?

Reply:
There were inconsistencies within this file so I openned #85.

Reply:
Does this go far enough to have support for SUSE Linux 12? For SUSE Linux 11? What else is needed to get SUSE Linux in the installation documentation?
Thanks.